driver/src/main/scala/api/commands/AggregationFramework.scala

```diff
@@ -522,4 +522,40 @@ trait AggregationFramework[P <: SerializationPack]
         elms.result()
       }))))
   }
+
+  // Change Stream
+  //
+  // Specification:
+  // https://github.com/mongodb/specifications/blob/master/source/change-streams/change-streams.rst#server-specification
+
+  /**
+   * Low level pipeline operator which allows to open a tailable cursor against subsequent ChangeEvents of a given
+   * collection (since MongoDB 3.6).
+   * https://docs.mongodb.com/manual/reference/change-events/
+   *
+   * For common use-cases, you might prefer to use the `watch` operator on a collection.
+   *
+   * Note: the target mongo instance MUST be a replica-set (even in the case of a single node deployement).
+   *
+   * @param resumeAfter the id of the last known Change Event, if any. The stream will resume just after that event.
+   * @param startAtOperationTime the operation time before which all Change Events are known. Must be in the time range
+   *                             of the oplog. (since MongoDB 4.0)
+   * @param fullDocument if set to UpdateLookup, every update change event will be joined with the *current* version of the
+   *                     relevant document.
+   */
+  final class ChangeStream(
+    resumeAfter: Option[pack.Value] = None,
+    startAtOperationTime: Option[pack.Value] = None, // TODO restrict to something more like a timestamp?
```

Comment:
What's expected by the command? A bson timestamp? If yes we can require a `Long` ? Or a `T: Numeric` ?

Reply:
If it can be opaque with responsibility to the dev/caller to provide something appropriate (doc). In this case would rename the arg without time which tend to "force" the expected semantic/type, maybe `offset` or a name like that.

Reply:
I tried to keep the name the same as in the official API, so its semantics are implied by the general mongo doc.